sdk/core/Azure.Core/tests/HttpPipelineMessageTest.cs

```diff
@@ -3,6 +3,8 @@
 
 using System;
```

Comment:
Looks like I accidentally moved this file to Identity in an overzealous reshuffling of shared source files!